ingest_python_scripts/launch_nara.py

```
@@ -0,0 +1,266 @@
+
+
+def stage_copy_to_ec2(month):
+    step(1, f"Copy ZIP(s) from s3://dpla-hub-nara/raw_ingest_files/{month}/ to EC2")
+    s3_src   = f"{NARA_S3_BUCKET}/{month}/"
+    dest_dir = f"{NARA_ORIGINALS}/{month}"
+    print(f"  Source:      {s3_src}")
+    print(f"  Destination: {dest_dir}")
+
+    ssm_run(f"mkdir -p {dest_dir}", timeout_seconds=30)
+
+    print(f"  Syncing ZIPs to EC2 (this may take a few minutes)...")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)